src/javascript/JContent/JContent.accordion-items.jsx

```diff
@@ -80,10 +93,12 @@ export const jContentAccordionItems = registry => {
         label: 'jcontent:label.contentManager.navigation.pages',
         defaultPath: siteKey => '/sites/' + siteKey,
         getPathForItem: node => {
-            const pages = node.ancestors.filter(n => n.primaryNodeType.name === 'jnt:page');
+            const pages = node.ancestors
+                .filter(n => n.primaryNodeType.name === 'jnt:page')
+                .sort(compareNodesByPathDesc);
             return pages[pages.length - 1].path;
         },
-        canDisplayItem: node => !filesRegex.test(node.path) && !contentsRegex.test(node.path),
+        canDisplayItem: (selectionNode, folderNode) => selectionNode ? !filesRegex.test(selectionNode.path) && !contentsRegex.test(selectionNode.path) : folderRegex.test(folderNode.path),
```

Comment:
if only one of the 2 params is expected, maybe use `({selectionNode, folderNode})` instead ?